### PR TITLE
 feat(prebuilt): add rabbitmq:3-management

### PR DESCRIPTION
**Pull request by sudoskys**
RabbitMQ is a widely used open-source message broker that implements the Advanced Message Queuing Protocol (AMQP)

https://github.com/rabbitmq/rabbitmq-server
https://www.rabbitmq.com/rabbitmqctl.8.html

### CODE REVIEW

**Review by MichaelYuhe — APPROVED** [review]
LGTM, thanks for your contribution! cc @pan93412 @yuaanlin for further review

File: prebuilts/rabbitmq.toml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#:schema ./schema.json
+# 待审查
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:2]:
```suggestion
```

File: prebuilts/rabbitmq.toml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+id = "rabbitmq"
+name = "RabbitMQ"
+
+# TODO Download svg icon from https://www.svgrepo.com/svg/303576/rabbitmq-logo
+icon = "https://raw.githubusercontent.com/zeabur/service-icons/main/marketplace/rabbitmq.svg"
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:8]: Just send a PR to here 😄 It is public → https://github.com/zeabur/service-icons

File: prebuilts/rabbitmq.toml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+[[ports]]
+id = "database"
+port = 5672
+type = "TCP"
+
+[[volumes]]
+id = "data"
+# mountPath = "/var/lib/rabbitmq"
+dir = "/data/rabbitmq" # TODO
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:23]: Not sure what you want to do. You should fill `/var/lib/rabbitmq` if RabbitMQ saves persistent files there.

File: prebuilts/rabbitmq.toml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+
+
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:26]:
```suggestion
```

File: prebuilts/rabbitmq.toml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+[env]
+RABBITMQ_DEFAULT_USER = { default = "admin" } # or guest?
+RABBITMQ_DEFAULT_PASS = { default = "${PASSWORD}" }
+
+RABBITMQ_DEFAULT_VHOST = { default = "/" }
+
+RABBITMQ_HOST = { default = "${CONTAINER_HOSTNAME}", expose = true, readonly = true }
+RABBITMQ_PORT = { default = "${DATABASE_PORT}", expose = true, readonly = true }
+
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:35]:
```suggestion
```

File: prebuilts/rabbitmq.toml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+RABBITMQ_CONNECTION_STRING = { default = "amqp://${RABBITMQ_DEFAULT_USER}:${RABBITMQ_DEFAULT_PASS}@${RABBITMQ_HOST}:${RABBITMQ_PORT}", expose = true, readonly = true }
+RABBITMQ_URI = { default = "${RABBITMQ_CONNECTION_STRING}", expose = true, readonly = true }
+
+[[instructions]]
+type = "TEXT"
+title = "Command to connect to your RabbitMQ"
+# TODO RabbitMQ cli is not wide used,check https://serverfault.com/a/961291 https://stackoverflow.com/a/56140362
+content = "rabbitmqadmin --host ${PORT_FORWARDED_HOSTNAME} --port ${DATABASE_PORT_FORWARDED_PORT} --username ${RABBITMQ_DEFAULT_USER} --password ${RABBITMQ_DEFAULT_PASS} --vhost ${RABBITMQ_DEFAULT_VHOST} list queues"
+showWhen = "PORT_FORWARDING"
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:45]: This is not a necessary section. If nobody uses it, you can just remove this part.

```suggestion
```

File: prebuilts/rabbitmq.toml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+[[instructions]]
+type = "TEXT"
+title = "RabbitMQ username"
+content = "${RABBITMQ_DEFAULT_USER}"
+category = "Credentials"
+
+
```
Comment (pan93412) [inline prebuilts/rabbitmq.toml:53]:
```suggestion
```

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: prebuilts/rabbitmq.toml (hunk 2 above)
Comment (sudoskys) [reply prebuilts/rabbitmq.toml:8]: Done

**Comment by sudoskys** [comment]
A new change is pushed

File: prebuilts/rabbitmq.toml (hunk 3 above)
Comment (sudoskys) [reply prebuilts/rabbitmq.toml]: https://www.rabbitmq.com/backup.html

**Review by sudoskys — COMMENTED** [review]
Review

**Review by pan93412 — APPROVED** [review]

**Pull request merged**